Actions/CreateApp/AppHelper.psm1

```diff
@@ -0,0 +1,136 @@
+<#
+This module contains some useful functions for working with app manifests.
+#>
+
+. (Join-Path -path $PSScriptRoot -ChildPath "..\AL-Go-Helper.ps1" -Resolve)
+$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+$alTemplatePath = Join-Path -Path $here -ChildPath "AppTemplate" 
+
+
+$validRanges = @{
+    "PTE"           = "50000..99999";
+    "AppSource App" = "100000..$([int32]::MaxValue)";
+    "Test App"      = "50000..$([int32]::MaxValue)" ;
+};
+
+function Confirm-IdRanges([string] $templateType, [string]$idrange ) {  
+    $validRange = $validRanges.$templateType.Replace('..', '-').Split("-")
+    $validStart = [int](stringToInt($validRange[0]))
```

Comment:
What is stringToInt?
I don't have that function and I cannot find a definition.
You can just use [int]$validRange[0] as PowerShell does implicit conversion